### PR TITLE
Doc 8454 add sizing information to server docs

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

**Comment by RayOffiah** [comment]
Found one or two more places where 'events' was mentioned

**Review by shivaniguptacb — APPROVED** [review]

File: modules/install/pages/sizing-general.adoc [hunk 1]
```diff
@@ -215,12 +215,14 @@ Based on the above formula, these are the suggested sizing guidelines:
 This tells you that the RAM requirement for the whole cluster is 7 GB.
 Note that this amount is in addition to the RAM requirements for the operating system and any other software that runs on the cluster nodes.
 
-==== CPU Overhead
+[#cpu-overhead]
+== CPU Overhead
```
Comment (osfameron) [inline modules/install/pages/sizing-general.adoc:219]: Just wanted to check that promoting to h2 is deliberate, as it's not mentioned in ticket spec.

File: modules/install/pages/sizing-general.adoc [hunk 2]
```diff
@@ -215,12 +215,14 @@ Based on the above formula, these are the suggested sizing guidelines:
...
 
 When sizing, you must account for raw CPU overhead when using a high number of buckets. 
 
-We recommend that you set aside 0.2 core/bucket for operational stability. This overhead does not account for any front-end workload which must be sized separately.
+* We recommend that you set aside 0.2 core/bucket for operational stability. This overhead does not account for any front-end workload which must be sized separately.
 
+* Monitoring is recommended for CPU usage and System Limits
```
Comment (osfameron) [inline modules/install/pages/sizing-general.adoc]: Link to monitoring might be useful.

**Review by osfameron — APPROVED** [review]
👍 
2 minor comments.

File: modules/install/pages/sizing-general.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/install/pages/sizing-general.adoc:219]: Yep, that's fine. It brings it in line with the other headings.

File: modules/install/pages/sizing-general.adoc (hunk 2 above)
Comment (RayOffiah) [reply modules/install/pages/sizing-general.adoc]: Good idea.

**Pull request merged**